test/compile/mark/bar.test.ts

```diff
@@ -100,4 +100,40 @@ describe('Mark: Bar', function() {
       });
     });
   });
+  
+  describe('ranged bar', function() {
+    it('vertical bars should work with aggregate', function() {
+      const model = parseUnitModel({
+        "data": { "url": "data/population.json" },
+        "mark": "bar",
+        "encoding": {
+          "x": { "field": "age", "type": "ordinal" },
+          "y": { "field": "people", "aggregate": "q1", "type": "quantitative"},
+          "y2": { "field": "people", "aggregate": "q3", "type": "quantitative"}
+        }
+      });
+
+      const props = bar.properties(model);
+      assert.deepEqual(props.xc, { scale: 'x', field: 'age'});
+      assert.deepEqual(props.y, { scale: 'y', field: 'q1_people' });
+      assert.deepEqual(props.y2, { scale: 'y', field: 'q3_people' });
+    });
+    
+    it('vertical bars should work with aggregate', function() {
```

Comment:
Incorrect description of what you're testing:  'vertical bars should work with aggregate'

This seems like horizontal